### PR TITLE
release: Add `v1.4.0` and `v1.3.1` release notes

### PR DESCRIPTION
**Pull request by marclop**
## Description

Adds the current release and the previous (branched) `v1.3.1` release notes.

### CODE REVIEW

File: docs/release_notes/v1.3.1.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_darwin_amd64.tar.gz[ecctl_1.3.1_darwin_amd64.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_32-bit.deb[ecctl_1.3.1_linux_32-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_32-bit.rpm[ecctl_1.3.1_linux_32-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_386.tar.g[ecctl_1.3.1_linux_386.tar.g]
```
Comment (rhass) [inline docs/release_notes/v1.3.1.adoc:19]: Is this suppose to be `.gz`?

File: docs/release_notes/v1.4.0.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/1.4.0/ecctl_1.4.0_darwin_amd64.tar.gz[ecctl_1.4.0_darwin_amd64.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.4.0/ecctl_1.4.0_linux_32-bit.deb[ecctl_1.4.0_linux_32-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.4.0/ecctl_1.4.0_linux_32-bit.rpm[ecctl_1.4.0_linux_32-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.4.0/ecctl_1.4.0_linux_386.tar.g[ecctl_1.4.0_linux_386.tar.g]
```
Comment (rhass) [inline docs/release_notes/v1.4.0.adoc:19]: Same question here.

**Review by rhass — CHANGES_REQUESTED** [review]
I validated the URLs, and the `.g` files return a 404, changing it to `.gz` worked. 

Eg:

```
 rhass@kean  ~/src/cloudy-cloud   main ●  curl -IL https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_386.tar.g
HTTP/2 404
date: Tue, 22 Jun 2021 21:54:38 GMT
content-length: 0
strict-transport-security: max-age=15724800; includeSubDomains
via: 1.1 google
alt-svc: clear

 rhass@kean  ~/src/cloudy-cloud   main ●  curl -IL https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_386.tar.gz
HTTP/2 200
date: Tue, 22 Jun 2021 21:54:40 GMT
content-type: application/x-tar
content-length: 5315556
last-modified: Wed, 14 Apr 2021 04:37:24 GMT
etag: "e5a54e67c7f9d6ebff7ce5f93af785cb"
accept-ranges: bytes
cache-control: public, max-age=600
strict-transport-security: max-age=15724800; includeSubDomains
via: 1.1 google
alt-svc: clear
```

File: docs/release_notes/v1.3.1.adoc (hunk 1 above)
Comment (marclop) [reply docs/release_notes/v1.3.1.adoc:19]: Yeah it is. The template must be broken then

**Comment by marclop** [comment]
Thanks Ryan, I fixed the links, they're good to go now.

File: scripts/changelog.tpl.adoc [hunk 3]
```diff
@@ -16,7 +16,7 @@ Download the release binaries:
 https://download.elastic.co/downloads/ecctl/VERSION_REPLACE/ecctl_VERSION_REPLACE_darwin_amd64.tar.gz[ecctl_VERSION_REPLACE_darwin_amd64.tar.gz]
 https://download.elastic.co/downloads/ecctl/VERSION_REPLACE/ecctl_VERSION_REPLACE_linux_32-bit.deb[ecctl_VERSION_REPLACE_linux_32-bit.deb]
 https://download.elastic.co/downloads/ecctl/VERSION_REPLACE/ecctl_VERSION_REPLACE_linux_32-bit.rpm[ecctl_VERSION_REPLACE_linux_32-bit.rpm]
-https://download.elastic.co/downloads/ecctl/VERSION_REPLACE/ecctl_VERSION_REPLACE_linux_386.tar.g[ecctl_VERSION_REPLACE_linux_386.tar.g]
+https://download.elastic.co/downloads/ecctl/VERSION_REPLACE/ecctl_VERSION_REPLACE_linux_386.tar.g[ecctl_VERSION_REPLACE_linux_386.tar.gz]
```
Comment (nordbergm) [inline scripts/changelog.tpl.adoc:19]: Shouldn't `ecctl_VERSION_REPLACE_linux_386.tar.g` still be `ecctl_VERSION_REPLACE_linux_386.tar.gz`? Looks like we just change the link title?

**Review by nordbergm — CHANGES_REQUESTED** [review]

File: scripts/changelog.tpl.adoc (hunk 3 above)
Comment (marclop) [reply scripts/changelog.tpl.adoc]: 😅

**Review by nordbergm — APPROVED** [review]
LGTM

**Pull request merged**